### PR TITLE
Add note

### PR DESCRIPTION
**Pull request by MuskanChhatrasal**
FUNCTIONALITY ADDED:
Add note functionality 
Archive functionality
Restore from archive functionality
Delete from archive functionality

UI will be doing later

### CODE REVIEW

File: src/Pages/Archived/archived.js [hunk 1]
```diff
@@ -1,13 +1,43 @@
...
+  const {archivedNotes} = authState;
+  
+  const {restoreFromArchive, deleteFromArchive} = useArchive();
   return (
-    <div className='archived-container'>
-        <Sidebar />
-        <h1>Archive</h1>
+    <>
+    <Sidebar />
+    {archivedNotes.length === 0 ? <h1 style={{marginTop: '-35rem', marginLeft: '40rem'}}>No Archived</h1>
```
Comment (Developers-codz) [inline src/Pages/Archived/archived.js:17]: inline styling can be removed

File: src/Pages/Home/home.css [hunk 2]
```diff
@@ -6,7 +7,7 @@
   margin-left: 20rem !important;
   height: 15rem;
   margin-top: 4rem !important;
-  width: 35rem;
+  /* width: 35rem; */
```
Comment (Developers-codz) [inline src/Pages/Home/home.css:10]: remove commented code

**Review by Developers-codz — APPROVED** [review]
Everything looks good
Nothing much to suggest just few observations

File: src/Pages/Archived/archived.js [hunk 3]
```diff
@@ -1,13 +1,43 @@
...
+      :
+      archivedNotes.map((note)=>{
+      return (
+         <div className='archived-container' style={{marginTop: '-46.5rem', marginBottom: '40rem'}}>
```
Comment (aayushjain-code) [inline src/Pages/Archived/archived.js:21]: Try  less Use Of Inline styling

File: src/index.js [hunk 4]
```diff
@@ -5,16 +5,24 @@ import App from "./App";
...
 import {BrowserRouter} from 'react-router-dom'
 import { AuthProvider } from "./Context/authContext";
+import { NoteProvider } from "./Context/noteContext";
+import { ArchiveProvider } from "./Context/archivedContext";
+
 
 // Call make Server
 makeServer();
 
 ReactDOM.render(
```
Comment (aayushjain-code) [inline src/index.js:15]: Try to use Prettier Extension for Formatting For better Understanding

**Review by aayushjain-code — APPROVED** [review]
Everything Looks fine few Suggestions added Go Ahead and merge it!!!

**Comment by MuskanChhatrasal** [comment]
> Everything looks good Nothing much to suggest just few observations

Okayy. Will implement it. Thankyou!!

**Comment by MuskanChhatrasal** [comment]
> Everything Looks fine few Suggestions added Go Ahead and merge it!!!

Thankyou for your suggestions will take care of it.

**Pull request merged**